docs/detections/value-list-exceptions.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[value-lists-exceptions]]
+== Create and manage value lists
+
+Value lists hold multiple values of the same Elasticsearch data type, such as IP addresses, which are used to determine when an exception prevents an alert from being generated. You can use value lists to define exceptions for detection rules; however, you cannot use value lists to define endpoint rule exceptions. 
+
+Value lists are lists of items with the same {es} {ref}/mapping-types.html[data type]. You can create value lists with these types:
+
+* `Keywords` (many {ecs-ref}/ecs-field-reference.html[ECS fields] are keywords)
+* `IP Addresses`
+* `IP Ranges`
+* `Text`
+
+After creating value lists, you can use `is in list` and `is not in list` operators to <<add-exceptions,define exceptions>>.
+
+TIP: You can also use a value list as the <<indicator-value-lists,indicator match index>> when creating an indicator match rule.
+
+[float]
+[[create-value-lists]]
+=== Create value lists
+When you create a value list for a rule exception, be mindful of the list's size and data type. All rule types support value list exceptions, but extremely large lists or certain data types have limitations. 
+
+Custom query, machine learning, and indicator match rules support the following value list types and sizes:
+
+* **Keywords** or **IP addresses** list types with more than 65,536 values
+* **IP ranges** list type with over 200 dash notation values (for example, `127.0.0.1-127.0.0.4` is one value) or more than 65,536 CIDR notation values
+
+To create a value list:
+
+. Prepare a `txt` or `csv` file with all the values you want to use for
+determining exceptions from a single list. If you use a `txt` file, new lines
+act as delimiters.
++
+[IMPORTANT]
+=========================
+* All values in the file must be of the same {es} type.
+
+* Wildcards are not supported in value lists. Values must be literal values.
+
+* The maximum accepted file size is 9 million bytes.
```

Comment:
I think this is the only relevant update to add to 7.17 - the other parts here are not as they apply to functionalities that were added in 8.x. 

@e40pud could you confirm this as well. I think your [backport](https://github.com/elastic/kibana/pull/176379) would only add this these new constraints:

- file size
- file type

Reply:
My changes actually added `file type` constraints. `file size` was already added in `7.9.0`, `7.10.0` and `8.0.0` here https://github.com/elastic/kibana/pull/72898, though we did not have docs for that.

Reply:
Thanks, both! It looks like I just need to update [this section](https://www.elastic.co/guide/en/security/7.17/detections-ui-exceptions.html#manage-value-lists) by adding the file size limitation for value lists (9 million bytes). The supported file types are already mentioned in step 1 of that section.

Reply:
Fixed in [ce921a0](https://github.com/elastic/security-docs/pull/5251/commits/ce921a032be55ddce327c62e9b403c9c29cc56a5).